diff --unified --recursive --text srhtctl/usr/share/zsh/site-functions/_srhtctl srhtctl.new/usr/share/zsh/site-functions/_srhtctl
--- srhtctl/usr/share/zsh/site-functions/_srhtctl	2020-02-26 21:01:40.000000000 +0100
+++ srhtctl.new/usr/share/zsh/site-functions/_srhtctl	2020-02-26 21:03:14.348616825 +0100
@@ -66,6 +66,7 @@
 
 function _srhtctl_build_deploy {
   _arguments
+    '_files'
 }
 
 function _srhtctl_build_info {
@@ -122,6 +123,7 @@
     '(-e --expiration)'{-e,--expiration}'[Paste expiration in days]:' \
     '(-n --name)'{-n,--name}'[Name for paste file]:' \
     '(-v --visibility)'{-v,--visibility}'[Paste visibility]:'
+    '_files'
 }
 
 function _srhtctl_paste_delete {
